crates/rust/src/lib.rs

```diff
@@ -377,9 +372,14 @@ impl InterfaceGenerator<'_> {
             "
                 #[allow(clippy::all)]
                 pub mod {snake} {{
+                    #[used]
+                    #[doc(hidden)]
+                    #[cfg(target_arch = \"wasm32\")]
+                    static __FORCE_SECTION_REF: fn() = crate::__link_section;
```

Comment:
Should this use a relative path of some form instead of the crate root to handle `generate!` in submodules?

Reply:
Oh good point, yeah, `super` should do, I think.